app/controllers/coronavirus_form/able_to_leave_controller.rb

```diff
@@ -20,12 +20,24 @@ def submit
       render controller_path
     else
       update_session_store
+      write_responses
```

Comment:
Interesting, I was thinking of moving this to the show method of the results controller.

So instead of writing on the last question, we'd write everytime we load/refresh the results page.

Reply:
I think the show method wouldn't necessarily be the right place, since we'll be making a GET request.  Maybe the results page should be loaded on a POST request, then it would seem acceptable to write to the database there.

Reply:
I think this is the best we're going to be able to come up with tonight.

In most forms there would be a check-your-answers page or something, but this one doesn't have that, so last page it is.